pkgs/applications/networking/irc/weechat/default.nix

```diff
@@ -85,7 +92,7 @@ let
           on https://nixos.org/nixpkgs/manual/#sec-weechat .
         '';
         license = lib.licenses.gpl3;
-        maintainers = with lib.maintainers; [ lovek323 ];
+        maintainers = with lib.maintainers; [ lovek323 ncfavier ];
```

Comment:
Might as well prune lovek323. Last contribution to this package in 2014.

Reply:
Done. Maybe this could be done treewide for retired contributors?

Reply:
Well I guess retired committer ≠ retired contributor necessarily.

Reply:
I think we need an RFC to define the maintainer role in general and then we can aim for treewide action. I have discussed this many times with my local peers.